native/explorer/src/encoding.rs

```diff
@@ -493,6 +506,7 @@ pub fn list_from_series(s: ExSeries, env: Env) -> Result<Term, ExplorerError> {
         DataType::Date => date_series_to_list(&s, env),
         DataType::Time => time_series_to_list(&s, env),
         DataType::Datetime(time_unit, None) => datetime_series_to_list(&s, *time_unit, env),
+        DataType::Duration(time_unit) => duration_series_to_list(&s, *time_unit, env),
```

Comment:
Just a note that whatever we decide above we need to mirror here. :)

Reply:
Addressed by: [98a8ff7](https://github.com/elixir-explorer/explorer/pull/683/commits/98a8ff75d1114250d52d319f233c72cbd187d92d)